frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx

```diff
@@ -70,7 +70,7 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
       [],
     );
     const patches = containers.reduce((patch, container, i) => {
-      if (_.isEmpty(container.env)) {
+      if (_.isEmpty(container.envFrom)) {
```

Comment:
Line 82
`path: /spec/template/spec/containers/${i}/envFrom/-` => `path: /spec/template/spec/containers/${i}/envFrom`
Line 77
`value: [configMapRef],` => `value: configMapRef`
WDYT @afreen23 @gnehapk  ?

Reply:
The reason for this check is if `envForm` is present then we should add it to the existing array of secrets otherwise create a new array.

Reply:
Yeah I know that I wrote it. I am talking about lines 82 and 77.  Where we are having that extra `-`. Please read the whole comment @afreen23

Reply:
my bad /lgtm